src/components/DownloadContactListButton.tsx

```diff
@@ -8,12 +8,11 @@ type DownloadButtonProps = {
 export function DownloadContactListButton (props: DownloadButtonProps) {
     let downloadUrl = null
     let csvData = '"Property Address","Property Type (Marketing)","Owner","Owner Mobile","Owner Phone","Contact","Contact Mobile","Contact Work Phone"\r\n'
-
-    const objectArr = props.results
+    const arrayOfObjectsFromCRM = props.results
```

Comment:
I recommend `arrayOfPropertyObjects`. It gives some context as to what the objects are.